app/client/components/RegionFocusSwitcher.ts

```diff
@@ -408,6 +353,17 @@ export class RegionFocusSwitcher extends Disposable {
     return this.focusRegion({type: 'panel', id: 'right'}, {initiator: {type: 'cycle'}});
   }
 
+  private _canTabThroughMainRegion(use: UseCBOwner) {
+    const gristDoc = this._gristDocObs ? use(this._gristDocObs) : null;
+    if (!gristDoc) {
+      return true;
+    }
+    if (gristDoc) {
+      use(gristDoc.activeViewId);
```

Comment:
Nitpick: this feels peculiar in a function that is supposed to return a boolean. It almost appears mutative? I am not familiar with the code though, so feel free to ignore my comment if it doesn’t make sense. :)

Reply:
it's the framework's way to subscribe to observables ;)